test/kb_quast_server_test.py

```diff
@@ -267,6 +293,21 @@ def test_quast_app(self):
                                                  'workspace_name': self.ws_info[1]})[0]
         self.check_quast_app_output(ret, 315170, 315200, '6aae4f232d4d011210eca1965093c22d',
                                     '2010dc270160ee661d76dad6051cda32')
+    
+    @patch.object(kb_quast, "check_large_input", return_value=True)
```

Comment:
Unless I'm missing something check_large_input is always mocked and so never actually tested that a large input will result in a return of True. IMO at least one test should exercise the method for each of the two possible outputs.